### PR TITLE
Keras nightly pip

### PR DESCRIPTION
**Pull request by sampathweb**
* Adds Keras Nightly GitHub Action to upload to `keras-nightly`
* Adds options `--nightly` and `--rc` to `pip_build.py`
* Adds a dummy readme file to `kokoro` folder so GitHub shows the top level folder only in its view.

### CODE REVIEW

File: pip_build.py [hunk 1]
```diff
@@ -189,15 +208,34 @@ def install_whl(whl_fpath):
     os.system(f"pip3 install {whl_fpath} --force-reinstall --no-dependencies")
 
 
+def validate_rc(rc):
+    rc = rc.lower()
+    if rc[:2] != "rc" or not rc[2:].isdigit():
+        raise argparse.ArgumentTypeError(
+            f"--rc value {rc} should be of format rc[0-9]+."
```
Comment (fchollet) [inline pip_build.py:215]:
```
Invalid value for argument `--rc`. Expected a number in the range [0-9]. Received: rc={rc}
```

**Review by fchollet — COMMENTED** [review]
Nice work, thank you!

File: pip_build.py [hunk 2]
```diff
@@ -189,15 +208,34 @@ def install_whl(whl_fpath):
     os.system(f"pip3 install {whl_fpath} --force-reinstall --no-dependencies")
 
 
+def validate_rc(rc):
```
[the remark this answers is not in the document]
Comment (sampathweb) [reply pip_build.py]: Changed `rc` to `int` and used `rc_index` internally.

File: pip_build.py (hunk 1 above)
Comment (sampathweb) [reply pip_build.py]: Changed `rc` to `type=int` and used the default validation for `int` thus removing the need for `validate_rc` method.

**Review by fchollet — APPROVED** [review]
LGTM, thank you

**Pull request merged**